Userland/Utilities/beep.cpp

```diff
@@ -1,13 +1,15 @@
 /*
- * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2022, the SerenityOS developers.
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibMain/Main.h>
 #include <unistd.h>
 
-int main()
+ErrorOr<int> serenity_main(Main::Arguments)
 {
-    sysbeep();
+    if (sysbeep() < 0)
+        return Error::from_errno(errno);
```

Comment:
Considering `sysbeep()` is a syscall wrapper, this should be `Error::from_syscall("beep", rc)`.
I believe the normal way to do this is to add a `Core::System::beep()` wrapper and call it here with `TRY()`